parm/config/config.vrfy

```diff
@@ -16,26 +16,24 @@ export CDFNL="gdas"               # Scores verification against GDAS/GFS analysi
 
 export MKPGB4PRCP="YES"           # Make 0.25-deg pgb files in ARCDIR for precip verification
 export VRFYFITS="YES"             # Fit to observations
-export VSDB_STEP1="YES"           # Populate VSDB database
-export VSDB_STEP2="NO"
-export VRFYG2OBS="YES"            # Grid to observations, see note below if turning ON
-export VRFYPRCP="YES"             # Precip threat scores
-export VRFYRAD="YES"              # Radiance data assimilation monitoring
-export VRFYOZN="YES"              # Ozone data assimilation monitoring
-export VRFYMINMON="YES"           # GSI minimization monitoring
+export VSDB_STEP1="NO"            # Disable VSDB
+export VSDB_STEP2="NO"            # Disable VSDB
+export VRFYG2OBS="NO"             # Grid to observations, see note below if turning ON
+export VRFYPRCP="NO"              # Precip threat scores
+export VRFYRAD=${VRFYRAD:-"YES"}  # Radiance data assimilation monitoring
+export VRFYOZN=${VRFYOZN:-"YES"}  # Ozone data assimilation monitoring
+export VRFYMINMON=${VRFYMINMON:-"YES"} # GSI minimization monitoring
 export VRFYTRAK="YES"             # Hurricane track verification
 export VRFYGENESIS="YES"          # Cyclone genesis verification
-export VRFYFSU="NO"               # Cyclone genesis verification (FSU)
-export RUNMOS="NO"                # whether to run entire MOS package
+export VRFYFSU="YES"               # Cyclone genesis verification (FSU)
+export RUNMOS="YES"                # whether to run entire MOS package
```

Comment:
Can you please explain why the defaults are changed?
Is it intended that `dev_v16` on R&D machines will run with these new defaults?

Reply:
Tracker code manager Jiayi requested the VRFYFSU be turn on.
Will reverse this change to the original.

Reply:
Ok. And what about he other defaults; e.g. 
```
VSDB_STEP1
VRFYG2OBS
VRFYPRCP
RUNMOS
```
Is there a need for providing external over-rides for these?
```
VRFYRAD
VRFYOZN
VRFYMINMON

```

Reply:
The new ecflow run verification job VRFYRAD, VRFYOZN, and VRFYMINMON differently. I adopt the same solution here. Doing so is to test and ensure job run as designed.
I will revert it.